docs/user-guide/secrets/index.md

```diff
@@ -778,7 +778,8 @@ Pod level](#use-case-secret-visible-to-one-container-in-a-pod).
 
 ### Risks
 
- - In the API server secret data is stored as plaintext in etcd; therefore:
+ - If the API server runs with argument "--storage-media-type=application/json", secret data is stored as
+   plaintext in etcd; therefore:
```

Comment:
I'm not sure that --storage-media-type makes a difference. It's still plain text, just surrounded by random bytes. Also technically it's not plaintext, it's base64 but that's quite irrelevant. Can we leave this part as is?